templates/btax/results.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'btax/input_base.html' %}
+{% load staticfiles %}
+{% load flatblocks %}
+{% block style %}
+{{block.super}}
+<link href="{% static 'css/vendor/bootstrap3-block-grid.min.css' %}" rel="stylesheet">
+<link href="text/javascript" src="{% static 'js/vendor/DataTables/datatables.min.css' %}"></link>
+<style>
+  .btn {
+    color: black;
+  }
+  .text-white {
+    color: white;
+  }
+  .table > thead > tr > th {
+    border: 0;
+  }
+  .single-line {
+    white-space: nowrap;
+  }
+</style>
+{% endblock %}
+{% block content %}
+<div class="wrapper">
+  {% include 'taxbrain/header.html' %}
+  <div class="result-header">
+    <div class="result-header-control">
+      <h1>{% flatblock "taxbrain_results_header" %}</h1>
+    </div>
+    <p class="meta">These results were generated on {{ created_on|date:"D, M jS Y \a\t g:iA" }} using B-Tax <a href="#">version {{ unique_url.btax_vers }}</a> and Taxcalc <a href="#">version {{ unique_url.taxcalc_vers }}</a> TaxBrain. (ID: 10045)</p>
+    {% if is_behavior %}
```

Comment:
this section can be deleted. many of the "if is_*" section are meant to help customize the TaxBrain results page. We don't have that issue with BTax results.